app/src/main/java/org/p2p/wallet/common/ui/widget/ActionButtonsView.kt

```diff
@@ -56,7 +60,17 @@ class ActionButtonsView @JvmOverloads constructor(
                     listener?.onActionButtonClicked(ActionButton.SWAP_BUTTON)
                 }
             }
+            viewActionSell.apply {
+                textViewButtonTitle.setText(R.string.home_sell)
+                imageButtonButtonIcon.setImageResource(R.drawable.ic_action_sell)
+                imageButtonButtonIcon.setOnClickListener {
+                    listener?.onActionButtonClicked(ActionButton.SEND_BUTTON)
+                }
+            }
         }
+
+        val actionButtonsIsGone: Map<ActionButton, Boolean> = ActionButton.values().associateWith { false }
+        setActionButtonsIsVisible(buttonsToIsVisible = actionButtonsIsGone)
```

Comment:
weird code here and looks like it does nothing.
1. Where do you pass the feature toggle if the `SELL_BUTTON` should be shown?
2. maybe `visibleActionButtons`
3. `setActionButtonsVisible(areVisible: Boolean)`

Reply:
@gslevinkov what about this?